### PR TITLE
src: add hatsune miku command

### PR DESCRIPTION
**Pull request by cthpw103**
please mayo please

## What does this PR do?
- [x] Adds a feature
- [ ] Fixes a bug
- [ ] Edits documentation
- [ ] Edits this repository
- [ ] Something else

## Things to check
- [x] Have you tested changes locally?
- [x] Have you followed our linter rules?

## Notes
- [ ] New dependencies added
- [ ] Unsure of functionality

## Description
MIKU IS COOL

### CODE REVIEW

**Comment by espimarisa** [comment]
Not sure why node-gyp throws a fit half of the time, but that's unrelated. Fix your linting (view the commit) and I'll merge this evening.

**Comment by cthpw103** [comment]
done

File: src/commands/Miscellaneous/miku.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  async run(msg) {
+    const body = await fetch("https://safebooru.org/index.php?page=dapi&s=post&q=index&json=1&tags=hatsune_miku%20rating:safe")
+      .then(res => res.json().catch(() => {}));
+
+    if (!body || !body[0].image || !body[0].directory) {
+      return this.bot.embed("❌ Error", "No images were found.", msg, "error");
+    }
+
+    if (body[0].image.endsWith(".webm") || body[0].image.endsWith(".mp4")) {
+      return this.bot.embed("❌ Error", `Post is a video. You can view it [here](${body[0].image}).`, msg, "error");
```
Comment (TTtie) [inline src/commands/Miscellaneous/miku.js:23]: Why is this an error? This should be an informational/warning embed instead.

**Review by TTtie — CHANGES_REQUESTED** [review]

File: src/commands/Miscellaneous/miku.js (hunk 1 above)
Comment (espimarisa) [reply src/commands/Miscellaneous/miku.js:23]: The other commands have it called ❌ Error.

This reminds me we should probably be more diverse in our messages and not just say "something bad happened oh no"

**Pull request merged**